packages/core/src/renderer/components/entity-settings/extension-registrator.injectable.ts

```diff
@@ -36,6 +39,7 @@ export interface RegisteredEntitySetting {
   components: EntitySettingComponents;
   source?: string;
   group: string;
+  isShown: IComputedValue<boolean>;
```

Comment:
I wonder if this should be optional to not break something..

Reply:
The `EntitySettingRegistration` from extension has optional `visible`. When the extension settings are read the settings are converted to `RegisteredEntitySetting` and then we can make this not optional which is nicer for the users of `RegisteredEntitySetting`